### DISCUSSION TITLE
plugin_music with Spotify does not play

### DISCUSSION THREAD

**Discussion by Minghao2812** [lowlighter/metrics#896] [Q&A] [answered]
Hi @lowlighter and all the contributors to this project! Thanks for bringing such an impressive tool for creating dynamic profile pages! My plugin_music with Spotify does not play my song when I visit my profile page, and I'm not sure if I setup it wrong. My profile page looks like this:
![image](https://user-images.githubusercontent.com/57082435/154449981-20ceead7-ad03-403f-bf60-9f4e3e4cb83a.png)

And the Action code looks like this:
```
## plugin: music
plugin_music: yes
plugin_music_provider: spotify
plugin_music_token: "***************************************, ****************************************, *******************************************************************"
plugin_music_mode: playlist
plugin_music_playlist: https://open.spotify.com/embed/playlist/*****************************
```

Would you or any contributors like to kindly help me with this?

Many thanks,

Minghao

**Comment by lowlighter** [comment, marked as answer]
Hi 👋 !
Thanks for your support !

>  My plugin_music with Spotify does not play my song when I visit my profile page

I'm not sure I understood this part 😅 
Do you expect the music to play and auto-start when someone goes on your profile ? If that's the case then it's not possible, this plugin is just to display what you listen to or show your favorites track to other people so they have a glimpse of your music taste 🙂 

Else it seems your setup looks fine to me 👍 . The token part is optional if you're specifying a playlist instead though

**Reply by spenserblack** [reply]
This got me curious since I never considered it, and I stumbled on [this SO question](https://stackoverflow.com/q/14412027). It would be something fun to experiment with. I can't believe it's possible, but I might test it out for fun some time to make sure. I am *not* saying that metrics should pursue an attempt to support audio.

**Reply by lowlighter** [reply]
I think it may work in raw `<svg>`, but if it's embed through an `<img>` tag I'd say it doesn't.

Additionally, even if it's supported through `<img>`, I'm pretty sure GitHub flavored markdown would remove these but who knows, maybe it has never been attempted before x) 

But yeah, anycase I don't think it's the kind of feature we'd support directly 😅

**Reply by spenserblack** [reply]
Yeah, I'd be *very* surprised if this was ever possible in a GH README. I just meant for `<svg>` in general.

**Reply by Minghao2812** [reply]
Thanks for your reply! Yeah, I thought it was an auto-play plugin, just like some web blogs do. Then I tested it several times to make sure it won't be too loud for a new visitor, but it turns out no sound at all 🤣 Now I understand that!
